### ISSUE TITLE
Some problem when used ini config

### ISSUE THREAD

**Issue by lrf739146825** [libretro/mame#484]
When "Read Configuration" and "MAME ini Path" are enabled in "Quick Menu" - "Core Options" - "System". Add the mame.ini file to the RetroArch/system/name/ini/.
1. I found that RetroArch can only "Load Content", which is located in the rompath configuration path of the mame.ini file.
2. When Mame runs the game, I change the game through RetroArch's "Load Content", and RetroArch will crash directly.
RetroArch Log : [libretro WARN] [LUA] emu.register_start is deprecated - please use emu.add_machine_reset_notifier

**Comment by sonninnos** [comment]
Yes, the rom path in the ini will override everything. And that LUA thing sounds like you need to keep the system plugins dir up-to-date.

**Comment by lrf739146825** [comment]
I downloaded mame0265b_64bitxe and installed it. Copy the plugins folder to Retroarch/system/name/plugins. Set the value of the pluginspath attribute to system/name/plugins in the mame.ini file.The problem has not been resolved. 
I configure the interface menu of MAME to display in Chinese and use MAME's plugin functions through INI. This requires the use of uismall.bdf fonts that support CJK. I have uploaded the Libertro: MAME configuration kit that I am currently using. This may help you find the reason for the crash.
[RetroArch-MAME_Config.zip](https://github.com/user-attachments/files/15508276/RetroArch-MAME_Config.zip)

**Comment by lrf739146825** [comment]
Description correction  /system/name/plugins  -> /system/mame/plugins

**Comment by lrf739146825** [comment]
After checking one by one, I found that the problematic plugin is "gdbstub". Remove this plugin from the plugins folder, and then use RetroArch's "Load Content" to replace the loaded game and it will no longer crash.
